specification/authorization/resource-manager/Microsoft.Authorization/2017-10-01-preview/authorization.json

```diff
@@ -0,0 +1,766 @@
+            "name": "parentResourcePath",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The parent resource identity.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "resourceType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource type of the resource.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource to get the permissions for."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of permissions.",
+            "schema": {
+              "$ref": "#/definitions/PermissionGetResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetConfigurations": {
+            "$ref": "./examples/GetResourcePermissions.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Authorization/providerOperations/{resourceProviderNamespace}": {
+      "get": {
+        "tags": [
+          "ProviderOperationsMetadata"
+        ],
+        "operationId": "ProviderOperationsMetadata_Get",
+        "description": "Gets provider operations metadata for the specified resource provider.",
+        "parameters": [
+          {
+            "name": "resourceProviderNamespace",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace of the resource provider."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The API version to use for the operation."
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "resourceTypes",
+            "description": "Specifies whether to expand the values."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns the operations metadata.",
+            "schema": {
+              "$ref": "#/definitions/ProviderOperationsMetadata"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetConfigurations": {
+            "$ref": "./examples/GetProviderOperationsRP.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Authorization/providerOperations": {
+      "get": {
```

Comment:
question for @ravbhatnagar
Is this allowed? The linter marks an error on this swaggers because they are not providing an operatin that lists all available operations in the API. I found this other operation that looks like does exactly that. Is it mandatory to use the exact same path that is provided by the linter error?

Reply:
No, this is fine for this RP. For all other RPs, the path that linter points is correct.